spec/unit/helper_spec.rb

```diff
@@ -68,6 +70,23 @@
     end
   end
 
+  describe ".content_is_binary?" do
+    use_given_filesystem
+
+    it "detects xml content as non-binary" do
+      expect(Machinery::content_is_binary?(File.read(given_file("xml_file")))).to be(false)
+      expect(Machinery::content_is_binary?(File.read(given_file("xml_file_2")))).to be(false)
```

Comment:
In which way is the test with `xml_file_2` different from the one with `xml_file`?